ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+as you intended.
+
+It takes only a single configuration mistake on a single device to create havoc.
+
+Since operating many networking devices also typically result in having many networking engineers, 
+there is even more room for error. Different people do things differently and repetitive
+tasks are not always executed in the exact same manner.
+
+With Ansible AVD, you define not only the topology of your network centrally, but also which services are used where in a central repository
+of text files.
+
+The “source of truth” exists inside Ansible AVD at any point in time, meaning you have a full overview of your entire designed
+network configuration without having to look at individual network devices.
+An additional benefit of Ansible AVD is that by design you have an always available backup of your network configuration
+including automated documentation in markdown format.
+No longer do you have to remember to update documentation about which interface is connected to which device etc.,
+whenever you change something on a device.
+It's all done automatically based on the configuration built and applied to the network devices by Ansible, everytime you execute the playbooks.
+
+Below you will find two examples of documentation automatically created by Ansible AVD:
+
+- [5 stage CLOS fabric across two Data Centers](../molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md)
+- [A single leaf inside the fabric above](../molecule/eos_designs-twodc-5stage-clos/documentation/devices/DC1-POD1-L2LEAF1A.md)
+
+## How do I use Ansible AVD
+
+Below are a number of links to additional documentation about Ansible AVD and Ansible in general:
+- With [eos_design](../../roles/eos_cli_config_gen)
+- With [eos_cli_config_gen](../../roles/eos_cli_config_gen)
+- With [Ansible Tower](https://docs.ansible.com/ansible/2.3/tower.html)
+- With CI/CD
+- With [eos_config_deploy_cvp](../../roles/eos_config_deploy_cvp) / [eos_config_deploy_eapi](../../roles/eos_config_deploy_eapi)
+
+Ansible AVD comes with pre-built templates that you can either decide to re-use as-is or adapt to your liking.
+
+Once the templates reflect your desired network configuration, you deploy the configuration either directly to the Arista EOS devices or via Arista Cloudvision using its internal change management system, where the changes can be reviewed, approved and executed in accordance with existing company requirements.
+The deployment is executed typically using CLI commands or via the GUI available with Ansible Tower. 
+
+For example, running the CLI command: 
+```
+ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml --tags build
+```
+...will execute the playbook `playbooks/dc1-fabric-deploy-cvp.yml` with the tag `build`. 
+
+Once the configuration has been deployed, an automated validation of the designed configuration versus the running configuration can be executed using the included `validation` playbook, to ensure that:
```

Comment:
Refer to the role as ``eos_validate_state``. Will also be interesting to link an example.